### PR TITLE
Check version

### PR DESCRIPTION
**Pull request by tavangar79**

### CODE REVIEW

File: .github/actions/build/action.yml [hunk 1]
```diff
@@ -41,6 +41,31 @@ inputs:
 runs:
   using: "composite"
   steps:
+    - name: Parse pubspec.yaml for version
```
Comment (manjav) [inline .github/actions/build/action.yml:44]: move out check-version from build action into the new action

**Review by manjav — CHANGES_REQUESTED** [review]

File: .github/actions/build/action.yml (hunk 1 above)
Comment (tavangar79) [reply .github/actions/build/action.yml]: done

Comment (manjav) [reply .github/actions/build/action.yml]: android and iOS start building

Comment (tavangar79) [reply .github/actions/build/action.yml]: i just added it to Release Tag after checked and confirmed by you we can add to other files

**Pull request merged**